go.mod

```diff
@@ -109,4 +109,5 @@ replace (
 	github.com/gorilla/mux => github.com/containous/mux v0.0.0-20181024131434-c33f32e26898
 	github.com/mailgun/minheap => github.com/containous/minheap v0.0.0-20190809180810-6e71eb837595
 	github.com/mailgun/multibuf => github.com/containous/multibuf v0.0.0-20190809014333-8b6c9a7e6bba
+	golang.org/x/sys => golang.org/x/sys v0.0.0-20210817190340-bfb29a6856f2
```

Comment:
This will pin it to this exact version. Wondering, does it work if you add golang.org/x/sys to "require" with an `// indirect`, to force a more current version?

Reply:
@thaJeztah you are right: the replace directive must not be used to upgrade/pin a version, the right way is to use `indirect`